### PR TITLE
created my account

### PR DESCRIPTION
**Pull request by claudiap0**
added a new page on the website

### CODE REVIEW

**Comment by mihai-vlc** [comment]
the changes look good

File: src/my_account.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+def account():
+    pass
```
Comment (mihai-vlc) [inline src/my_account.py]: can we add a new line to this file ?

Comment (claudiap0) [reply src/my_account.py]: yes

**Review by mihai-vlc — APPROVED** [review]

**Comment by StefanOprea30** [comment]
good work! XD

**Pull request merged**